### PR TITLE
Unit tests - get_rich_users 

### PR DESCRIPTION
**Pull request by martinnenseth**
#75 
- Inserts test data 
- Checks if the actual rich users is the excpected result
- Checks if the bot is a rich user
- Checks if not rich users is not in get_rich_users.

### CODE REVIEW

**Review by Zhiftie — APPROVED** [review]

File: BuffBot/dbUnitTest.py [hunk 1]
```diff
@@ -6,23 +6,38 @@
...
-        # Create a Database object we can call functions on.
+        self.emptyTable('game_restriction')
+        self.emptyTable('members')
+        # Create a Database object to call the relevant functions.
         self.db = Database()
         # Insert dummy data into the database.
         self.db.flag_gaming_channel("1", "Hearthstone", 1)
+        self.db.insert_coins(85431603408420864, 4000, 'Lenny#1112')
+        self.db.insert_coins(235892294857916417, 23, 'Christian#1111')
+        self.db.insert_coins(278286021903515656, 20000, 'BuffBot#0334')  # Mocking the bot user.
```
Comment (Cmoen11) [inline BuffBot/dbUnitTest.py:34]: Is it possible to undo these changes after the test? Like self.remove_coins after the tests?

Comment (martinnenseth) [reply BuffBot/dbUnitTest.py:34]: Yes, that is possible. However, if we would like to wipe all the changes made by the tests to the database, including the emtpytable(table), we would have to  querry all the actual data in the relevant testing tables: 
image


, save this result to a variable in setUp(). Then we would have to querry the content of this variable back into the DB in a tearDownClass function. 
In cases where there is a lot of data in these tables, I imagine this could eat a huge amount of RAM, _but do we care?_. 

There are of course other options, which may be much better than this concept I provided. If so, do tell! 
Could it be an idea to simply have an own branch designated for testing?
